*** crossroad-0.7/src/crossroad.py	Thu Jun  8 22:32:32 2017
--- crossroad-0.7-new/src/crossroad.py	Fri Oct 27 17:14:38 2017
***************
*** 536,542 ****
          # which may overwrite some variables. So instead I set my own bashrc,
          # where I make sure to first run the user rc files.
          bashrc_path = os.path.join(install_datadir, 'crossroad/scripts/shells/bash/bashrc.' + available_platforms[target].name)
!         command = [shell, '--rcfile', bashrc_path]
      elif shell[-3:] == 'zsh':
          zdotdir = os.path.join(install_datadir, 'crossroad/scripts/shells/zsh.' + available_platforms[target].name)
          # SETUP the $ZDOTDIR env.
--- 536,542 ----
          # which may overwrite some variables. So instead I set my own bashrc,
          # where I make sure to first run the user rc files.
          bashrc_path = os.path.join(install_datadir, 'crossroad/scripts/shells/bash/bashrc.' + available_platforms[target].name)
!         command = [shell, '--rcfile', bashrc_path, '-ci', '"echo finished"']
      elif shell[-3:] == 'zsh':
          zdotdir = os.path.join(install_datadir, 'crossroad/scripts/shells/zsh.' + available_platforms[target].name)
          # SETUP the $ZDOTDIR env.
